### ISSUE TITLE
Distributions for parameters from {epiparameter} vs manual input in transmissibility report

### ISSUE THREAD

**Issue by CarmenTamayo** [epiverse-trace/episoap#109]
On PR #100 , {distcrete} and {epitrix} have been replaced by {epiparameter}, to discretise distributions and convert summary statistics into distribution parameters, respectively.

Now the issue is that `epiparameter::discrete` requires an epidist object, which is currently not generated when users provide parameters manually, rather than through the epiparameter database. This also affects the syntax of the plotting function, which is different when using an epidist object too.

The solution would be either to revert to using {distcrete} or to convert the manually added parameters and their distribution to an epidist object.

thoughts? @Bisaloo

**Comment by Bisaloo** [comment]
I'm not sure.

Discretizing has side effects I didn't anticipate when this code was written. I thought as distcrete as just a nice interface and didn't realize it was actually an approximation of the continuous distribution.

As the same time, epidist objects at the moment require that you pass a disease name which we don't necessarily know. I don't know if it's an issue or if it would be okay to pass "Disease X".

**Comment by CarmenTamayo** [comment]
Could we add the disease name to the params list? as it is for epiparameter_disease?

**Comment by CarmenTamayo** [comment]
So we could avoid adding an extra parameter if there was one called "disease_name" or something like that, which were used both to select the disease from the database and to discretise/plot the si?

**Comment by Bisaloo** [comment]
Okay, let's try this and see how it goes when updating the report.

**Comment by CarmenTamayo** [comment]
I would like to check it out to compare, this issue is probably resolved now that we've agreed to create epidists even for the case where the user doesn't extract the parameters from the epiparameter database, but still I'd like to see the plotting method and compare

**Comment by CarmenTamayo** [comment]
@joshwlambert could I ask what the status of this plotting method is?

> Would speeding up merging the upcoming plotting method for `<epidist>` into `main` be helpful here? It produces a very similar plot to the current {episoap} plot and can be modified as it returns a ggplot2 object to include the title and custom theme. It might also simplify writing the code as the plot method will work independent of whether the distribution is continuous or discretised (only difference is whether the plot is a smooth curve or bars).

**Comment by joshwlambert** [comment]
It is still on a [feature branch](https://github.com/epiverse-trace/epiparameter/tree/epidist_plot). I will work on this ASAP in order to merge into the main branch for testing. Will let you know once it is ready.

**Comment by CarmenTamayo** [comment]
thank you @joshwlambert
